dev-libs/libmspack/libmspack-0.6~alpha.recipe

```diff
@@ -13,27 +13,43 @@ version of any particular compressor or decompressor, use the \
 mspack_version() function. The UNIX library interface version is defined as \
 the highest-versioned library component."
 HOMEPAGE="https://www.cabextract.org.uk/libmspack/"
-COPYRIGHT="2000-2015 Stuart Caie"
+COPYRIGHT="2000-2017 Stuart Caie"
 LICENSE="GNU LGPL v2"
-REVISION="2"
+REVISION="1"
 SOURCE_URI="$HOMEPAGE/libmspack-${portVersion/\~/}.tar.gz"
-CHECKSUM_SHA256="8967f275525f5067b364cee43b73e44d0433668c39f9376dfff19f653d1c8110"
+CHECKSUM_SHA256="1edbee82accb28e679ab538f803aab7a5a569e4102ccf1715b462b1bd915f921"
 SOURCE_DIR="libmspack-${portVersion/\~/}"
 
+SOURCE_FILENAME_2="libmspack-${portVersion/\~/_}-fix-tests.patch"
+srcGitRev_2="831379877f8c646c5643dd98305afdb5a6b4ca6c"
+SOURCE_URI_2="https://gitweb.gentoo.org/repo/gentoo.git/plain/dev-libs/libmspack/files/$SOURCE_FILENAME_2?id=$srcGitRev_2#noarchive"
+CHECKSUM_SHA256_2="ab1bbe6b469316c320507f1b13aee36fe06b3732949c52b8e5e0f8dd4c3cf8a1"
+
 ARCHITECTURES="x86_gcc2 x86 x86_64"
 SECONDARY_ARCHITECTURES="x86_gcc2 x86"
 
+libVersion="0.1.0"
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+
 PROVIDES="
 	libmspack$secondaryArchSuffix = $portVersion
-	lib:libmspack$secondaryArchSuffix = 0.1.0 compat >= 0
+	lib:libmspack$secondaryArchSuffix = $libVersionCompat
 	"
+if [ -z "$secondaryArchSuffix" ]; then
+PROVIDES="$PROVIDES
+	cmd:cabrip = $portVersion
+	cmd:chmextract = $portVersion
+	cmd:msexpand = $portVersion
+	cmd:oabextract = $portVersion
+	"
+fi
```

Comment:
I believe we should also have a policy for when should we just get rid of the `cmd:`s

Reply:
Well, I believe we do have such a policy. It's only that it's not documented ;)
From what I have understood, here are the rules:
* If a recipe builds on all primary architectures then it is recommended to drop the `cmd:` on secondary arch builds.
* If, however, the execution speed is important or if the runtimes could have unnoticed bugs (when built with gcc2) then we keep the `cmd:` for all builds.